### ISSUE TITLE
Unable to select a default (host) option during the vm deployment

### ISSUE THREAD

**Issue by kiranchavala** [apache/cloudstack#7204]
##### ISSUE TYPE

 * Bug report

##### COMPONENT NAME

~~~
Component:UI
~~~

##### CLOUDSTACK VERSION

~~~
4.18 nightly
4:17.2

~~~

##### SUMMARY

Unable to select a default (host) option during the vm deployment

##### STEPS TO REPRODUCE
~~~
Steps to reproduce the issue 

1. Go to the vm deployment page

2. Select an existing  host in the cluster

3. Try to Go back to Default host option ( Empty)

The Ui doesn't allow to select the default 
![hostselection](https://user-images.githubusercontent.com/1401014/218456694-11909775-3969-4b20-9abc-0fae817771a5.png)



~~~

##### EXPECTED RESULTS

~~~
The Ui should allow selecting the default host 


##### ACTUAL RESULTS

~~~
The Ui is not allowing to select the default 
~~~
~~~

**Comment by kiranchavala** [comment]
@borisstoyanov please add the tag Component:ui

**Comment by kiranchavala** [comment]
The issue is also observed in 4.17.2 release

**Comment by nvazquez** [comment]
Hi @kiranchavala - I've created a fix on PR #7256, can you please check?

**Comment by DaanHoogland** [comment]
@kiranchavala can you verify #7256 solves the issue please?

**Comment by DaanHoogland** [comment]
closing as solved by #7256
